easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.4-rc2-sirius-CrayIntel-18.08-cuda-9.1.eb

```diff
@@ -0,0 +1,41 @@
+# created by Anton Kozhevnikov (CSCS)
+easyblock = "ConfigureMake"
+
+name = 'QuantumESPRESSO-SIRIUS'
+version = '6.4-rc2'
+cudaversion = 9.1
+versionsuffix = '-cuda-%s' % cudaversion
```

Comment:
The eb file name `QuantumESPRESSO-6.4-rc2-sirius-CrayIntel-18.08-cuda-9.1.eb` does not match with the according generated module name `QuantumESPRESSO-SIRIUS/6.4-rc2-CrayIntel-18.08-cuda-9.1`. 

If you wish to keep the current module name, you should rename the eb file to `QuantumESPRESSO-SIRIUS-6.4-rc2-CrayIntel-18.08-cuda-9.1.eb`.

Reply:
Done!